File: ios/RegisterAPIModule.m
//
//  register.m
//  StlApi
//
//  Created by Seongwoo Park on 2022/03/27.
//  Copyright © 2022 Facebook. All rights reserved.
//

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(RegisterAPIModule, NSObject)

RCT_EXTERN_METHOD(getNearbyDevice:(RCTPromiseResolveBlock)resolve
                  withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/RegisterAPIModule.swift
//
//  register.swift
//  StlApi
//
//  Created by Seongwoo Park on 2022/03/27.
//  Copyright © 2022 Facebook. All rights reserved.
//

@objc(RegisterAPIModule)
class RegisterAPIModule: NSObject, RCTBridgeModule {
    
    @objc
    static func moduleName() -> String! {
        return "register"
    }
    
    @objc
    static func requiresMainQueueSetup() -> Bool {
        return false
    }

    @objc(getNearbyDevice:withRejecter:)
    func getNearbyDevice(resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
        resolve([
            "type": "",
            "id": ""
        ])
    }
}
